CONTRIBUTING.md

```diff
@@ -73,17 +75,8 @@ and create a new Git version tag (for example `v0.13.0`). Make sure you follow t
 Then, create [a new GitHub release](https://github.com/zk-org/zk/releases) with a copy of
 the latest `CHANGELOG.md` entries and the binaries for all supported platforms.
 
-Binaries can be created automatically using `make dist-linux` and `make dist-macos`.
-
-Unfortunately, `make dist-macos` must be run manually on both an Apple Silicon and Intel
-chips. The Linux builds are created using Docker and
-[these custom images](https://github.com/zk-org/zk-xcompile), which are hosted via
-[ghcr.io within zk-org](https://github.com/orgs/zk-org/packages/container/package/zk-xcompile).
-
-This process is convoluted because `zk` requires CGO with `mattn/go-sqlite3`, which
-prevents using Go's native cross-compilation. Transitioning to a CGO-free SQLite driver
-such as [cznic/sqlite](https://gitlab.com/cznic/sqlite) could simplify the distribution
-process significantly.
+Binaries can be downloaded from the `build-binaries.yml` Actions run
+that was triggered by pushing the tag and uploaded to the release.
```

Comment:
We need to adjust that as well, I'd keep that open as a last step when we figured out everything else ;).

Reply:
adjusted the whole section a little as it needed more detail anyway I think. But will need to be looked at again later anyway after this pr is sorted